pkg/loadbalancer/proxy.go

```diff
@@ -31,45 +29,57 @@ type proxyConfigData struct {
 
 type data struct {
```

Comment:
your type name is bad and you should feel bad. :slightly_smiling_face:

Reply:
it is horrible , isn't it ? the problem is that I run out of ideas and end with the same names in all the indirection layers   😄

Reply:
replaced